libbeat/docs/processors-config.asciidoc

```diff
@@ -359,6 +360,22 @@ _Tencent Cloud_
 }
 -------------------------------------------------------------------------------
 
+_Alibaba Cloud_
+
```

Comment:
Let's change the wording because we don't want to take the text verbatim from the Alibaba docs. How about adding this before the example:

`This metadata is only available when VPC is selected as the network type of the instance.`

Or, assuming we are talking about an ECS instance here (apologies for my ignorance), this would be even better and less confusing for users:

`This metadata is only available when VPC is selected as the network type of the ECS instance.`